### PR TITLE
Setup kettle namespace

### PR DESCRIPTION
**Pull request by ameukam**
Ref: https://github.com/kubernetes/k8s.io/issues/787

Signed-off-by: Arnaud Meukam <ameukam@gmail.com>

### CODE REVIEW

**Comment by ameukam** [comment]
/assign @spiffxp 
cc @MushuEE

File: groups/groups.yaml [hunk 1]
```diff
@@ -374,6 +375,16 @@ groups:
...
 
+  - email-id: k8s-infra-rbac-kettle@kubernetes.io
+    name: k8s-infra-rbac-kettle
+    description: |
+      Grants access to the `namespace-user` role in the `kettle` namespace on the `aaa` cluster
+    settings:
+      ReconcileMembers: "true"
+      WhoCanViewMembership: "ALL_MEMBERS_CAN_VIEW" # required
+    members:
+      - k8s-infra-prow-oncall@kubernetes.io
```
Comment (spiffxp) [inline groups/groups.yaml:386]: @MushuEE isnt in this group AFAIK (I would be ok if we changed that though)

**Review by spiffxp — APPROVED** [review]
/approve
/lgtm
/hold
Remove when ready to deploy

**Comment by ameukam** [comment]
/hold cancel 
@MushuEE When you have some bandwidth, can you add yourself to this group ? 🙏🏾 
If you want to test access, see :  https://github.com/kubernetes/k8s.io/blob/main/running-in-community-clusters.md#access-the-cluster

**Pull request merged**